test/functional/wallet_dump.py

```diff
@@ -49,7 +49,7 @@ def read_dump(file_name, addrs, script_addrs, hd_master_addr_old):
                     # count key types
                     for addrObj in addrs:
                         if addrObj['address'] == addr.split(",")[0] and addrObj['hdkeypath'] == keypath and keytype == "label=":
-                            # a labled entry in the wallet should contain both a native address
+                            # a labeled entry in the wallet should contain both a native address
```

Comment:
Already reported in #12543.

Reply:
So should I revert this one or just do nothing?

Reply:
Revert or rebase on top of `master` now that #12543 is merged.

Reply:
Reverted. Not familiar with git rebase and such...